### DISCUSSION TITLE
Mocking Browsershot for faster tests | Rias.be

### DISCUSSION THREAD

**Discussion by giscus** [riasvdv/rias.be#539] [General]
# Mocking Browsershot for faster tests | Rias.be

Spatie's Browsershot is a wonderful package that uses puppeteer to render a web page to transform it into an image or PDF. Having this functionality covered by tests generally requires you to install puppeteer in your CI environment and can slow down your tests a lot.

https://rias.be/blog/mocking-browsershot-for-faster-tests

**Comment by Silver343** [comment]
Do you have any advice for mocking when using the html() method as it is a static function?

**Comment by FranciscoCaldeira** [comment]
``Use `code` in your Markdown file.``
